apps/sv/src/main/scala/org/lfdecentralizedtrust/splice/sv/automation/SvDsoAutomationService.scala

```diff
@@ -394,18 +394,20 @@ class SvDsoAutomationService(
       )
     )
 
-  localSequencerClientContext.flatMap(_.internalClientConfig).foreach { internalClientConfig =>
-    registerTrigger(
-      new LocalSequencerConnectionsTrigger(
-        triggerContext,
-        participantAdminConnection,
-        internalClientConfig.decentralizedSynchronizerAlias,
-        dsoStore,
-        internalClientConfig.sequencerInternalConfig,
-        config.participantClient.sequencerRequestAmplification,
-        config.domainMigrationId,
+  if (!config.bftSequencerConnection) {
```

Comment:
Wondering if it still makes sense to keep this here, can't we do the same thing from the validator app already and thus move all the logic to the validator app?

Reply:
It's not the same as for the validator because of timing: The LocalSequencerConnectionsTrigger will wait until the sequencer is actually fully working. The validator in single-SV config will try to connect to it immediately which could be before the sequencer is fully working and you might blow up on tombstones or some crap.

I would suggest to just keep it as is and try to delete the single-SV config once we have enough confidence that this one works.